updateManager2204.patch

```diff
@@ -0,0 +1,20 @@
+--- /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py.orig	2025-06-25 15:15:56.941726136 +0300
++++ /usr/lib/python3/dist-packages/UpdateManager/UpdateManager.py	2025-06-25 15:43:17.614605707 +0300
+@@ -27,7 +27,7 @@
+ from gi.repository import Gio
+ from gi.repository import GLib
+ from gi.repository import GObject
+-import uaclient.api.u.pro.packages.updates.v1 as ua
++# import uaclient.api.u.pro.packages.updates.v1 as ua
```

Comment:
In my case the crash was caused by a `ModuleNotFoundError` because the pro package was uninstalled.

Not sure if commenting out `self._get_ua_security_status()` is necessary (for me it doesn't seem to do anything, currently Update Manager works fine with or without `_get_ua_security_status`), but it's probably a good idea to skip it anyway, since it may use some disabled/removed functionality.